x-pack/plugins/apm/server/routes/services/get_service_instances/get_service_instances_transaction_statistics.ts

```diff
@@ -215,14 +174,14 @@ export async function getServiceInstancesTransactionStatistics<
         const { doc_count: count, key } = serviceNodeBucket;
         const serviceNodeName = String(key);
 
-        // Timeseries is returned when isComparisonSearch is true
+        // Timeseries is returned when includeTimeseries is true
         if ('timeseries' in serviceNodeBucket) {
           const { timeseries } = serviceNodeBucket;
           return {
             serviceNodeName,
             errorRate: timeseries.buckets.map((dateBucket) => ({
               x: dateBucket.key,
-              y: dateBucket.errorRate.value,
+              y: dateBucket.failures.doc_count / dateBucket.doc_count,
```

Comment:
Q: Is it safe to say that we won't risk dividing by 0 here and a few lines below?

Reply:
If I'm not mistaken **doc_count** will never be `0`.